cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255)
+    description = serializers.CharField(required=False)
```

Comment:
Why do you think this field shouldn't be required? Wouldn't it be confusing if you try to buy a movie ticket, for example, and are not able to see any description?